File: src/main/java/dev/iancmd/operationrelife/blocks/world/AbstractIceBlock.java
package dev.iancmd.operationrelife.blocks.world;

import net.minecraft.world.level.block.Block;
import net.minecraft.world.level.block.state.BlockBehaviour;
import net.minecraft.world.level.material.Material;
import net.minecraftforge.common.extensions.IForgeBlock;

public class AbstractIceBlock extends Block implements IForgeBlock {
  public AbstractIceBlock(Properties of) {
    super(BlockBehaviour.Properties.of(Material.ICE_SOLID).friction(10.0F));
  }
}

File: src/main/kotlin/dev/iancmd/operationrelife/blocks/world/AbstractIceBlock.kt
package dev.iancmd.operationrelife.blocks.world

import net.minecraft.world.level.block.Block
import net.minecraft.world.level.material.MapColor
import net.minecraftforge.common.extensions.IForgeBlock

@Suppress("UNUSED_PARAMETER")
open class AbstractIceBlock(of: Properties?) :
  Block(Properties.of().friction(10.0f).mapColor(MapColor.ICE)), IForgeBlock
